front/src/components/myCar/Footer.tsx

```diff
@@ -46,40 +61,53 @@ export const Footer = ({
             <Text typo="Body3_Regular" palette="DarkGray">
               트림
             </Text>
-            {footerInfo.name.map((item, idx) => (
-              <Text
-                typo={idx ? 'Body3_Regular' : 'Heading4_Bold'}
-                palette="Black"
-                key={`trimModel_${idx}`}
-              >
-                {item}
+            <Text typo={'Heading4_Bold'} palette="Black">
+              {myCarInfo.trim.type?.name}
+            </Text>
+            <Flex justify="flex-start">
+              <Text typo={'Body3_Regular'} palette="Black">
+                {Object.values(myCarInfo.trim)
+                  .map((trim, idx) =>
+                    idx !== 0 && trim !== null ? trim.name : null,
```

Comment:
```suggestion
 idx !== 0 && trim?.name
```
이렇게 하면 안되는 이유가 있을까요?

Reply:
허걱 생각 못했어요 감사합니다~!~!~!